wallets/react-wallet-v2/package.json

```diff
@@ -39,22 +39,24 @@
     "mnemonic-keyring": "1.4.0",
     "near-api-js": "^0.45.0",
     "next": "12.1.5",
+    "permissionless": "0.0.0-main.20231120T173047",
```

Comment:
I believe that everything included in this version should be available on latest one of permisionless by now, you might be able to use proper version here in that case.

Reply:
I tried updating yesterday and got signature mismatch errors with the same code that works 😵 
Will try again today, maybe I need to tweak something for it to work.